Dockerfile

```diff
@@ -28,5 +28,8 @@ RUN cd /pip && \
 	--no-deps -r requirements/docker.txt && \
     rm -r build cache
 
+# Need uglify for code minimization.
+RUN npm install
```

Comment:
Hmm, also where do npm deps end-up? You don't really want npm deps in the tree they should be above the tree in the container so that arch issues aren't a problem and there's no confusion by having npm deps in the tree that were put there with a different arch to the host. That is unless we can pull the front-end bits out completely. I'm guessing not if this is using jingo etc.